### ISSUE TITLE
Add dataset-level description option

### ISSUE THREAD

**Issue by tsalo** [bids-standard/pybids-reports#61]
I'm starting to work on [CuBIDS](https://github.com/PennLINC/cubids), which is useful for annotating variant acquisitions in a BIDS dataset (e.g., adding an `acq` entity if some runs of a task have X volumes and some have Y volumes in the dataset). It would be great to have an option in pybids-reports that generates a description that accounts for variants by specifically looking at the `acq` field and reporting _across_ subjects.

For example, let's say we have a dataset like this:

```
sub-01/
    func/
        sub-01_task-rest_acq-100volumes_bold.nii.gz
sub-02/
    func/
        sub-02_task-rest_acq-100volumes_bold.nii.gz
sub-03/
    func/
        sub-03_task-rest_acq-90volumes_bold.nii.gz
```

In the current version of `pybids-reports`, we would end up with two patterns in the `BIDSReport` report `Counter`: one that describes subjects 01 and 02, and a second one that describes subject 03. My proposal is a parameter in `BIDSReport` that would describe the whole dataset, treating the different `acq` values as different variants on the same task (assuming all other entities are the same).

Something like:

> For session 01:
>     MR data were acquired using a 3-Tesla Siemens Prisma MRI scanner.
>     In two subjects, one run of resting-state fMRI data were collected (<params>). Each run was XX minutes in length, during
which 100 functional volumes were acquired.
>     In one subject, one run of resting-state fMRI data were collected (<params>). Each run was XX minutes in length, during
which 90 functional volumes were acquired.

**Comment by Remi-Gau** [comment]
> treating the different acq values as different variants on the same task (assuming all other entities are the same)

yeah that was my thought when I started reading the beginning of your issue.
